src/editor/object_option.hpp

```diff
@@ -49,11 +49,13 @@ class Rectf;
 class TileMap;
 class Writer;
 
-class ObjectOption
+class BaseObjectOption
 {
 public:
-  ObjectOption(const std::string& text, const std::string& key, unsigned int flags);
-  virtual ~ObjectOption();
+  BaseObjectOption(const std::string& text, const std::string& key, unsigned int flags);
+  virtual ~BaseObjectOption()
+  {
+  }
```

Comment:
Put this in the `.cpp`, or remove it.

Reply:
I can't, otherwise compilation errors.

Reply:
Could you then put the `{}` right next to the declaration's line?

Reply:
`virtual ~BaseObjectOption() = default;` in .hpp